assets/js/modules/thank-with-google/components/settings/SettingsView.js

```diff
@@ -75,6 +79,29 @@ export default function SettingsView() {
 		return null;
 	}
 
+	let supporterWall;
+
+	if ( supporterWallSidebars === undefined ) {
+		supporterWall = <ProgressBar small />;
+	} else if ( supporterWallSidebars.length > 0 ) {
+		supporterWall = (
+			<p className="googlesitekit-settings-module__meta-item-data">
+				<DisplaySetting value={ supporterWallSidebars?.join( ', ' ) } />
+			</p>
+		);
+	} else {
+		supporterWall = (
+			<p className="googlesitekit-settings-module__meta-item-data">
+				<Link
+					href={ supporterWallURL }
+					className="googlesitekit-settings-module__cta-button"
+				>
+					{ __( 'Add Supporter wall', 'google-site-kit' ) }
```

Comment:
I realise this was simply moved, but I think this should be `Add supporter wall`, as we lowercase it elsewhere when not used as a title...

```suggestion
					{ __( 'Add supporter wall', 'google-site-kit' ) }
```